test/dates/accessors.jl

```diff
@@ -118,7 +109,20 @@ function test_dates2(from,to)
         end
     end
 end
-test_dates2(0,2100)
+
+# test hour, minute, second
+let h=0, mi=0, s=0, ms=0, us=0, ns=0
+    for h = 0:23, mi = 0:59, s = 0:59,
+        ms in (0,1,500,999), us in (0,1,500,999), ns in (0,1,500,999)
```

Comment:
The [new accessor test case](https://github.com/JuliaLang/julia/pull/12274/files#diff-2f3dfed48f169e03378a18a8e7db28d4R114) triggers the GC segfault. From what I've seen so far it seems that 32-bit systems have this issue when they perform a large amount of tests. Since this particular test generates 33177600 `@test` calls it appears to regularly trigger the problem.